test/hotspot/jtreg/compiler/c2/cmove/TestFPComparison2.java

```diff
@@ -30,10 +30,10 @@
  * @test
  * @bug 8358892 8357551
  * @summary The test is to trigger code path of BoolTest::ge/gt in C2_MacroAssembler::enc_cmove_cmp_fp
- * @requires os.arch == "riscv64"
  * @requires vm.debug
+ * @requires os.arch == "riscv64" | os.arch=="aarch64"
```

Comment:
One minor question: Could this work for `x64` as well?

Reply:
Nice catch!

Reply:
The first version of this pr did not have this `@requires`, but CI shows that IR verification failed, so I add this `@requires` for both riscv and aarch64 (check https://github.com/openjdk/jdk/pull/28702/commits/7848a1bf263ad9f498287f7b243601731b67b269)
Yes, it's good to have it run on x86, but I think it's worth anothe pr to investigate it. How do you think about it?